nnstreamer.ini.in

```diff
@@ -26,4 +26,9 @@ enable_use_gpu=@TORCH_USE_GPU@
 [nnfw-runtime]
 prioritize_tflite_ext=@NNFW_RUNTIME_PRIORITIZE@
 
+# Set framework priority about model file extension when automatically selecting framework for tensor filter.
+[framework-priority]
```

Comment:
Discussion:
group-name, which one is better? 
[filter] vs [framework-priority]

[filter] already exists, move to line 10 if [filter] is better.

Reply:
Okay, I will move it to [filter] group.